Cannot find module 'jest-cli'

**Describe the bug**
When creating a new angular 8 project and adding    
"@angular-builders/jest": "^8.0.1" 
"jest": "^24.8.0

 and trying to run a test I get the following error

```
Cannot find module 'jest-cli'
Error: Cannot find module 'jest-cli'
```

**To Reproduce**
Steps to reproduce the behavior:
1. create a new angular 8 project 
2. Add the jest  builder/dependency
3. update angular.json to the following
```
 "test": {
          "builder": "@angular-builders/jest:run",
          "options": {

          }
        },
```
3. run ng test
4. See error

**Expected behavior**
I expect it to work as it did in Angular 7.  

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Builder:**
 - The name of the builder
 - The version of the builder

**Libraries**
`    "@angular-devkit/build-angular": "~0.800.0",`

**Additional context**
if I  npm install jest-cli as a package independently it works.  Do we need to install both jest and jest-cli as dev dependencies?

--- a/packages/custom-webpack/package.json
+++ b/packages/custom-webpack/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@angular-builders/custom-webpack",
-  "version": "8.0.1",
+  "version": "8.0.2",
   "description": "Custom webpack builders for Angular build facade. Allow to modify Angular build configuration without ejecting it",
   "main": "dist/index.js",
   "files": [

--- a/packages/jest/src/index.ts
+++ b/packages/jest/src/index.ts
@@ -1,7 +1,7 @@
 import { BuilderContext, BuilderOutput, createBuilder } from "@angular-devkit/architect";
 import { experimental, normalize, Path, schema, json } from "@angular-devkit/core";
 import { NodeJsSyncHost } from "@angular-devkit/core/node";
-import { run } from 'jest-cli';
+import { run } from 'jest';
 import { from, Observable } from "rxjs";
 import { map } from "rxjs/operators";
 import { CustomConfigResolver } from "./custom-config.resolver";

